.github/workflows/main.yml

```diff
@@ -0,0 +1,74 @@
+name: Add files to draft release
+
+on: repository_dispatch
+
+jobs:
+  build_win:
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
```

Comment:
python 3.8? seems like a version bump to me...haven't we been using `3.7.2`?

Reply:
Ah I just used the latest one. I can switch to 3.7 but It won't really change anything.

Reply:
let's keep things the same until we verify locally that `3.8` works - we don't get any benefit by switching anyway (?)